docs/output-logstash.asciidoc

```diff
@@ -213,66 +193,83 @@ This setting can be used to _override_ the system trust store for verifying the
+[id="plugins-{type}s-{plugin}-ssl_keystore_password"]
+===== `ssl_keystore_password`
 
 * Value type is <<password,password>>
 * There is no default value for this setting.
-* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_truststore_path>>
-* {prohibit-ssl-disabled-effective}
+* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_keystore_path>>
+* Cannot be combined with configurations that disable SSL.
 
-Password for the <<plugins-{type}s-{plugin}-ssl_truststore_path>>
+Password for the <<plugins-{type}s-{plugin}-ssl_keystore_path>>
 
 [id="plugins-{type}s-{plugin}-ssl_truststore_path"]
 ===== `ssl_truststore_path`
 
 * Value type is <<path,path>>
 * There is no default value for this setting.
 * When present, <<plugins-{type}s-{plugin}-ssl_truststore_path>> is also required.
-* {prohibit-ssl-disabled-effective}
-* {prohibit-ssl-verification-mode-none}
+* Cannot be combined with configurations that disable SSL.
+* Cannot be combined with <<plugins-{type}s-{plugin}-ssl_verification_mode, `+ssl_verification_mode => none+`>>.
 
 A path to a JKS- or PKCS12-formatted truststore with which to validate the identity claims of the downstream input.
 The provided identity will be used if the downstream input enables <<plugins-{type}s-{plugin}-config-ssl-trust,SSL client authentication>>.
 
+[id="plugins-{type}s-{plugin}-ssl_truststore_password"]
+===== `ssl_truststore_password`
+
+* Value type is <<password,password>>
+* There is no default value for this setting.
+* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_truststore_path>>
+* Cannot be combined with configurations that disable SSL.
+
+Password for the <<plugins-{type}s-{plugin}-ssl_truststore_path>>
 
 [id="plugins-{type}s-{plugin}-ssl_verification_mode"]
 ===== `ssl_verification_mode`
 
 * Value type is <<string,string>>
-* There is only one currently-supported mode:
+* The supported modes are:
 ** `full`: verifies that a certificate provided by the client has an identity claim matching <<plugins-{type}s-{plugin}-host>>, is signed by a trusted authority (CA), is within its valid date range, and that the client has possession of the associated key.
 ** `none`: performs no validation of the presented certificate
 
 * The default value is `full`.
-* {prohibit-ssl-disabled-effective}
+* Cannot be combined with configurations that disable SSL.
 
 When <<plugins-{type}s-{plugin}-ssl_client_authentication>> causes a client to present a certificate, this setting controls how that certificate is verified.
 
 NOTE: Client identity is not typically validated using SSL because the receiving server only has access to the client's outbound-ip, which is not always constant and is frequently not represented in the certificate's subject or subjectAltNames extensions.
 For more information, see https://www.rfc-editor.org/rfc/rfc2818#section-3.1[RFC2818 § 3.2 (HTTP over TLS -- Client Identity)]
```

Comment:
Rephrase: the output plugin uses `ssl_verification_mode` for validating a downstream input, not a [connecting] client. The caveat about client identity is not needed in this context because the output _is_ the client.

```suggestion
When communicating over SSL, this setting controls how the downstream input's certificate is verified.
```